actions.go

```diff
@@ -114,6 +114,9 @@ func (c *Action) SetEnv(k, v string) {
 
 // SetOutput sets an output parameter.
 func (c *Action) SetOutput(k, v string) {
+	v = strings.ReplaceAll(v, "%", "%25")
```

Comment:
should we just URL-encode the whole thing?

Reply:
or perhaps the list is larger? https://github.com/actions/toolkit/blob/d69e699ab956dac3d3d25bb3c3feda23c1cb3a6c/packages/core/src/command.ts#L92-L106

Reply:
I agree that there are two more symbols to add but URL encoding the whole thing will escape a lot more characters I think? I suggest copying the JS/TS version and leaving its link for future reference in case its updated and someone faces the problem?

Reply:
SGTM - let me know when you want re-review 😄

Reply:
I just need a minute.